CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 0.5.2
+> Published 29 Oct 2025
+
+## New Features
+- Add `subtask` extension for non-graph agents similar to `subgraphWithTask` (#982)
+- Add MistralAI LLM Client (#622)
+
+## Improvements
+- Replace string content and attachments list in messages with a unified content parts list to make the API more flexible and preserve text/attachment parts order (#1004)
+- Add input and output attributes to the NodeExecuteSpan span in OpenTelemetry to improve observability [(KG-501)](https://youtrack.jetbrains.com/issue/KG-501/NodeExecuteSpan-span-should-include-input-output-attributes)
```

Comment:
Maybe, drop the ticket names from links:
`https://youtrack.jetbrains.com/issue/KG-501/NodeExecuteSpan-span-should-include-input-output-attributes` -> `https://youtrack.jetbrains.com/issue/KG-501`